packages/docs/versioned_docs/version-2.0.0-beta-3/welcome.md

```diff
@@ -2,7 +2,12 @@
 title: What is UniRep?
 ---
 
-# 👏 Welcome
+# Introduction
+
+
+UniRep is a private, non-repudiable data storage and retrieval system designed as a base layer for building custom and interoperable zk applications. It serves as a memory layer, allowing users to receive data, prove facts about their data, and store results of large proofs with strong privacy guarantees and constant computation complexity. 
+
+UniRep can be utilized for web2 authentication bridging and storing application-specific data such as user reputation, actions, and memberships. This enables seamless interoperability between applications using publicly available state roots for user proofs.
```

Comment:
I think not only for web2. we can just say

`UniRep can store application-specific data such as user reputation, actions, and memberships.`

Reply:
and let web2 authentication be another sample?

Reply:
and we don't have to specify *state roots*
we can just say

`using publicly available state for user proofs`